### ISSUE TITLE
Leaflet 1.2 compatibility

### ISSUE THREAD

**Issue by fox91** [socib/Leaflet.TimeDimension#114]
This plugin works for me with Leaflet 1.2 but in browser's console I see some warning about deprecated features.

For example:
```
Deprecated include of L.Mixin.Events: this property will be removed in future releases, please inherit from L.Evented instead. checkDeprecatedMixinEvents@http://localhost:32768/bower_components/leaflet/dist/leaflet-src.js:401:47
Class.extend@http://localhost:32768/bower_components/leaflet/dist/leaflet-src.js:329:3
@http://localhost:32768/bower_components/leaflet-timedimension/dist/leaflet.timedimension.src.withlog.js:25:20
@http://localhost:32768/bower_components/leaflet-timedimension/dist/leaflet.timedimension.src.withlog.js:17:2
```

Setup:
```
leaflet#1.2.0
leaflet-timedimension#1.0.7
```

I've not tested every features.

Andrea

**Comment by bielfrontera** [comment]
fixed in #121 (thanks @murb and @fox91)

**Comment by fox91** [comment]
Support for Leaflet 1.2 in `package.json` and `bower.json`?
